plugins/services/src/js/service-configuration/PodHealthChecksConfigSection.js

```diff
@@ -1,44 +1,32 @@
 import React from 'react';
 
 import ConfigurationMapTable from '../components/ConfigurationMapTable';
-import DurationValue from '../components/ConfigurationMapDurationValue';
-import Heading from '../../../../../src/js/components/ConfigurationMapHeading';
-import Section from '../../../../../src/js/components/ConfigurationMapSection';
+import ConfigurationMapDurationValue from '../components/ConfigurationMapDurationValue';
+import ConfigurationMapHeading from '../../../../../src/js/components/ConfigurationMapHeading';
+import ConfigurationMapSection from '../../../../../src/js/components/ConfigurationMapSection';
 import {getContainerNameWithIcon} from '../utils/ServiceConfigDisplayUtil';
-import ValueWithDefault from '../components/ConfigurationMapValueWithDefault';
+import ConfigurationMapValueWithDefault from '../components/ConfigurationMapValueWithDefault';
 
 const COMMON_COLUMNS = [
   {
     heading: 'Grace Period',
     prop: 'gracePeriod',
     render(prop, row) {
-      return (
-        <DurationValue
-          units="sec"
-          value={row[prop]} />
-      );
+      return <ConfigurationMapDurationValue units="sec" value={row[prop]} />;
```

Comment:
I would prefer one attribute per line for readability, but that's personal preference 😄